components/OlasTokenPage/UsagePieChart.jsx

```diff
@@ -25,25 +25,51 @@ const UsagePieChart = ({ epoch, split, loading }) => (
         <Verify url="https://etherscan.io/address/0xc096362fa6f4A4B1a9ea68b1043416f3381ce300#readProxyContract#F15" />
       </div>
       <p className="mb-8 text-slate-500">
-        Tokens are distributed to developers and bonders each epoch. Epochs
-        run roughly once a month.
+        Tokens are distributed to developers, stakers and bonders each epoch.
+        Epochs run roughly once a month.
```

Comment:
Each epochs runs roughly for a month.

Reply:
Would say `Each epoch runs for roughly one month.`